### PR TITLE
[Draft] Loud Headsets for Command.

### LINKED ISSUE #7119: Loud headsets for Heads of Staff
### Description

I am unsure of their name on /tg/, but they were headsets that made your radio chatter 2 times as bigger, thus making your messages more noticeable. (And more annoying)

### PR DESCRIPTION
**Pull request by MaxIsJoe**
WIP. closes #7119

### CODE REVIEW

File: UnityProject/Assets/Scripts/Items/Headset.cs [hunk 1]
```diff
@@ -4,15 +4,39 @@
 /// <summary>
 ///     Headset properties
 /// </summary>
-public class Headset : NetworkBehaviour
+public class Headset : NetworkBehaviour, ICheckedInteractable<HandApply>
```
Comment (ThatDan123) [inline UnityProject/Assets/Scripts/Items/Headset.cs:7]: Think you want HandActivate?

**Comment by corp-0** [comment]
I'm not convinced using speech modifiers is the way to go. This should only affect how text is displayed in chat.

**Comment by MaxIsJoe** [comment]
> I'm not convinced using speech modifiers is the way to go. This should only affect how text is displayed in chat.

Any suggestions on how to approach this then?

File: UnityProject/Assets/Scripts/Items/Headset.cs (hunk 1 above)
Comment (MaxIsJoe) [reply UnityProject/Assets/Scripts/Items/Headset.cs:7]: What's the difference? HandApply already seems to work

Comment (ThatDan123) [reply UnityProject/Assets/Scripts/Items/Headset.cs:7]: Well hand apply is using it on something, hand activate is clicking on it in hand.

**Comment by corp-0** [comment]
> Any suggestions on how to approach this then?

I would look for the code that sends text to chat and try to make it flexible so we can pass font size and style as parameters. This would also help to make chat more dynamic, like in ss13, where people screaming appear bigger in chat.

**Comment by MaxIsJoe** [comment]
Loudspeak PR is too old that it still uses the old version of Unity, starting from scratch.

**Pull request closed without merging**